Library/Formula/docker-compose.rb

```diff
@@ -0,0 +1,79 @@
+class DockerCompose < Formula
+  homepage "http://www.fig.sh/"
+  url "https://github.com/docker/compose/archive/1.1.0.tar.gz"
+  sha1 "175066934c19f455606b16f1b4e4b9f26fc3f599"
+
+  conflicts_with "fig", :because => "Branding change from Fig to Docker Compose."
+
+  bottle do
+    cellar :any
+    revision 1
+    sha1 "af332540bdc40e6e574d92ca7169c71071af9a5d" => :yosemite
+    sha1 "1f29c6947f4d6a797041dae82d0e5e2463224043" => :mavericks
+    sha1 "53a11121643db837d8d684859e9141643366f63c" => :mountain_lion
+  end
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "libyaml"
+  depends_on "docker" => :recommended
+  depends_on "boot2docker" => :recommended
+
+  resource "docker-py" do
+    url "https://pypi.python.org/packages/source/d/docker-py/docker-py-0.7.2.tar.gz"
+    sha1 "c526e95ea974b2a40392dbe00d0ecbfffa6c5d4b"
+  end
+
+  resource "pyyaml" do
+    url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
+    sha1 "1a2d5df8b31124573efb9598ec6d54767f3c4cd4"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha1 "d168e6d01f0900875c6ecebc97da72d0fda31129"
+  end
+
+  resource "dockerpty" do
+    url "https://pypi.python.org/packages/source/d/dockerpty/dockerpty-0.3.2.tar.gz"
+    sha1 "4b323b7c2fce84452dc5950ff1b32d97d9a412aa"
+  end
+
+  resource "texttable" do
+    url "https://pypi.python.org/packages/source/t/texttable/texttable-0.8.1.tar.gz"
+    sha1 "ca505fb7424defa99f5a85bdaf433a6b24ffa82c"
+  end
+
+  resource "docopt" do
+    url "https://pypi.python.org/packages/source/d/docopt/docopt-0.6.2.tar.gz"
+    sha1 "224a3ec08b56445a1bd1583aad06b00692671e04"
+  end
+
+  resource "requests" do
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.4.3.tar.gz"
+    sha1 "411f1bfa44556f7dd0f34cd822047c31baa7d741"
+  end
+
+  resource "websocket-client" do
+    url "https://pypi.python.org/packages/source/w/websocket-client/websocket-client-0.11.0.tar.gz"
+    sha1 "a38cb6072a25b18faf11d31dd415750692c36f33"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/docker-compose", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!